extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -500,6 +500,10 @@ export class AmpA4A extends AMP.BaseElement {
           // resolve it to a concrete value, but we'll lose track of
           // fetchResponse.headers.
           return fetchResponse.arrayBuffer().then(bytes => {
+            if (bytes.byteLength == 0) {
```

Comment:
Small comment here about what this case is and why this is the policy would be helpful.